ui/src/app/shared/utils.ts

```diff
@@ -128,5 +128,19 @@ export const Utils = {
 
     getCurrentNamespace(): string {
         return localStorage.getItem('current_namespace');
+    },
+
+    hasWarningCondition(conditions: WorkflowCondition[]): boolean {
```

Comment:
can we have less util functions? 
I think you should associate this with the `WorkflowCondition` class, eg. `WorkflowCondition.hasWarining(..)`